docs/source/build-images.md

```diff
@@ -1,12 +1,32 @@
 # Create a first release and image of your project
 
-This document describes how the [AICoE CI][1] and Thoth pipelines can be used to create the images, and where you can find the images for this project. For the purpose of this tutorial, the required images have already been created using these pipelines.
+This document describes how the [AICoE CI][1] and [Thoth](https://github.com/thoth-station) pipelines can be used to create the images for this project. **For the purpose of this tutorial, we have created the required images of the upstream repo. No action is required.** If you want to build these resources from your own fork, use the following instructions for image builds.
+
+## Set up AICoE CI and Thoth bots on your fork
+
+You will first need to configure your repo to use AICoE CI and Thoth. The [AICoE-CI tooling](https://github.com/AICoE/aicoe-ci) can be set up in just a few steps. Start by installing the AICoE CI GitHub application by [following this link](https://github.com/apps/aicoe-ci). When installing this application, select your profile for the organization, and specify the `elyra-aidevsecops-tutorial` as the repository.
+
+Once the application is installed, you will need to add Thoth's bots as collaborators. Navigate to your fork of `elyra-aidevsecops-tutorial`. Under the repository's **Settings**, go to **Manage Access** and click on "Invite a collaborator" and add Sesheta. Sesheta is a friendly Thoth bot who is used to help automate tasks. 
```

Comment:
It is sometimes confusing when the @sesheta invitation is not automatically accepted. Maybe we can add here, if the Sesheta invitation is not accepted, we can open an issue, reach out to someone on slack etc? @harshad16 @pacospace whats the best way to reach out to the team for pending invitations?
This could be a blocker for someone following through the tutorial otherwise.

Reply:
we can have them to open issue in the aicoe-ci.
we can create a default template and then try to automate that issue to resolve it as soon as possible.

Reply:
https://github.com/AICoE/aicoe-ci/issues/new?assignees=goern%2Charshad16&labels=area%2Fcyborgs%2Cbot%2Csig%2Fcyborgs&template=request_sesheta.yaml&title=Help+with+Sesheta+invite

this might help

Reply:
🎉 awesome! thanks @harshad16